Testreglar/254/Nett/nett-2.5.4a.json

```diff
@@ -0,0 +1,151 @@
+{
+	"id": "nett-2.5.4a",
+	"type": "web",
+	"kravTilSamsvar": "<ul>\r\n<li><span lang=\"NO-BOK\" style=\"font-size: 11.0pt; line-height: 107%; font-family: 'Arial',sans-serif; mso-fareast-font-family: Calibri; mso-fareast-theme-font: minor-latin; mso-bidi-font-family: 'Times New Roman'; mso-ansi-language: NO-BOK; mso-fareast-language: EN-US; mso-bidi-language: AR-SA;\">Funksjonalitet som kan brukes med å bevege enheten eller gjennom brukerbevegelser</span>, kan også brukes med vanlige brukergrensesnittkomponenter og</li>\r\n<li>Brukeren kan slå av bevegelsesaktivering av innhold via\r\n<ul>\r\n<li>innstillinger i brukeragenter som lar brukeren slå av bevegelsesaktivering på systemnivå eller</li>\r\n<li>en mekanisme for å slå av bevegelsesaktivering, og mekanismen ligger på samme side som innholdet med bevegelsesaktivering</li>\r\n</ul>\r\n</li>\r\n<li>Kravet gjelder med mindre\r\n<ul>\r\n<li>Støttet grensesnitt: Bevegelsen er brukt for å betjene funksjonalitet gjennom et grensesnitt som er tilgjengelighetsstøttende</li>\r\n<li>Avgjørende: Bevegelsen er avgjørende for funksjonaliteten, og det vil gjøre aktiviteten ugyldig.</li>\r\n</ul>\r\n</li>\r\n</ul>",
```

Comment:
Når muligheden for at slå av bevægelsesaktivering ligger på samme side, som funktionen, vil der være mulighed for at brugeren allerede har aktiveret bevægelsesaktivering ved en fejl, inden det lykkes at slå den fra.

Technique G213: "Provide conventional controls and an application setting for motion activated input" (https://www.w3.org/WAI/WCAG21/Techniques/general/G213) som er "sufficient" (og altså en måde at bestå 2.5.3 på) tilbyder to forskellige muligheder for at slå fra: 

- I Example 1: "Shake to undo can be turned off in a settings page."
- I Example 2: "...and a tick-box that prevents the motion from having an effect."

... ud over denne, der er nævnt i Understanding 2.5.3 (https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html): "Applications may be able to meet this requirement by supporting operating system settings which allow the user to disable motion detection at the system level."